src/routes/authRoutes.js

```diff
@@ -37,26 +40,26 @@ router.get('/callback', (req, res) => {
 });
 router.get(
 
-  "/auth/google",
-  passport.authenticate("google", { scope: ["email", "profile"] })
+  '/auth/google',
+  passport.authenticate('google', { scope: ['email', 'profile'] }),
 );
 router.get(
-  "/auth/google/callback",
-  passport.authenticate("google", {
+  '/auth/google/callback',
+  passport.authenticate('google', {
     session: false,
-    failureRedirect: "/",
+    failureRedirect: '/',
   }),
-  googleAuthHandler
+  googleAuthHandler,
 );
 
 router.post('/logout', logout);
```

Comment:
vendor route missed, add the two lines:
import vendor from '../controller/vendorController.js';

router.post('/vendor',isAdmin, vendor);